src/SelectPanel/SelectPanel.tsx

```diff
@@ -53,6 +56,8 @@ export function SelectPanel({
   renderAnchor = props => <DropdownButton {...props} />,
   anchorRef: externalAnchorRef,
   placeholder,
```

Comment:
Question, is this needed with `placeholderText` being available?

Reply:
The naming is confusing here. It looks like `placeholder` is what gets displayed when there are no results for a filter query. `placeholderText` is the placeholder for the filter input 😅

Reply:
Oh I see 😆, then yeah we should be good!